.github/workflows/docker.build.yml

```diff
@@ -56,4 +56,24 @@ jobs:
           cache-to: type=local,mode=max,dest=/tmp/.buildx-cache
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
-          labels: ${{ steps.docker_meta.outputs.labels }}
\ No newline at end of file
+          labels: ${{ steps.docker_meta.outputs.labels }}
+      #Create PR on deploy repo from a merged PR
+      -
+        name: create pull request for deploy on staging
+        uses: actions/checkout@v3
+        if: github.event.review.state == 'approved'
+        with:
+          repository: wmde/wbaas-deploy
+      # Make changes to pull request here
+      - run: |
+          get_tag=$(echo ${{ steps.docker_meta.outputs.tags }} | cut -d':' -f2)
+          yq -i '.image.tag=$get_tag' k8s/helmfile/env/local/ui.values.yaml.gotmpl
+          yq -i '.image.tag=$get_tag' k8s/helmfile/env/staging/ui.values.yaml.gotmpl
+        if: github.event.review.state == 'approved'
+      - uses: peter-evans/create-pull-request@v4
+        if: github.event.review.state == 'approved'
+        with:
```

Comment:
We will also need to set a unique `branch` value, else we'll get conflicts if two PRs are opened by this at the same time.

Reply:
I added a branch